src/screens/NewsScreen/Header.js

```diff
@@ -68,56 +65,56 @@ const styles = StyleSheet.create({
   avatar: {
     width: 24,
     height: 24,
-    alignSelf: 'center',
+    alignSelf: 'center'
   },
   container: {
     flexDirection: 'row',
     // paddingHorizontal: SIZES.base,
     // marginLeft: 12,
     paddingTop: 8.5,
     paddingBottom: 8.5,
-    display: 'flex',
+    display: 'flex'
   },
   domain: {
     lineHeight: 14.52,
     fontSize: 12,
     fontFamily: fonts.inter[600],
-    flexShrink: 1,
+    flexShrink: 1
   },
   domainNameContainer: {
     flexDirection: 'row',
-    alignItems: 'center',
+    alignItems: 'center'
   },
   domainPostDate: {
     lineHeight: 14.52,
     fontSize: 12,
     fontFamily: fonts.inter[400],
-    color: colors.gray,
+    color: COLORS.gray8
   },
-  wrapperItem: { backgroundColor: 'white', marginBottom: 16 },
+  wrapperItem: {backgroundColor: COLORS.white, marginBottom: 16},
   wrapperImage: {
     borderRadius: 45,
     borderWidth: 0.2,
     borderColor: 'rgba(0,0,0,0.5)',
     width: 46,
     height: 46,
     justifyContent: 'center',
-    alignItems: 'center',
+    alignItems: 'center'
   },
   image: {
     height: 46,
     width: 46,
-    borderRadius: 45,
+    borderRadius: 45
   },
   wrapperText: {
-    backgroundColor: 'white',
+    backgroundColor: COLORS.white,
     borderRadius: 8,
-    borderColor: '#00ADB5',
+    borderColor: COLORS.holyTosca,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value has been changed from `'#00ADB5'` to `COLORS.holyTosca`. Ensure that this change in color value is intentional and consistent with the overall theme of the application.
```diff
- borderColor: '#00ADB5',
+ borderColor: COLORS.holyTosca,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->